helm/rbac-operator/templates/rbac-tenant-admin.yaml

```diff
@@ -4,8 +4,7 @@ kind: ClusterRole
 metadata:
   name: tenant-admin
   labels:
-    app: rbac-operator
-    giantswarm.io/service-type: "managed"
```

Comment:
I see. I'd leave `_helpers.tpl` untouched as it's copy/pasteable at this point and leave this label as an extra one.

@corest do we need this label at all?

Reply:
No response, so I decided to roll back to common `_helpers.tpl`.

Reply:
You have my blessing :D